packages/@okta/vuepress-site/docs/concepts/brands/index.md

```diff
@@ -42,17 +42,24 @@ Okta Verify and the Okta plugin support multibrand. Use the following minimum ve
 - Okta Verify for macOS: MOV 3.10
 - Okta Verify for Windows: v3.10
 
-### Multibrand customizations and custom domains
+### Multibrand and custom domains
 
 You can create up to three custom domains with multibrand customizations. Increase your limit to 200 custom domains by contacting support.
 
 You can only visit a branded touchpoint (such as a logo or color) after you map to a custom domain. Create a brand and map it to a custom domain. Then, you can make further customizations, preview them, and publish them. See [Custom domains](/docs/guides/custom-url-domain/main/#about-okta-domain-customization).
 
-### About subdomain brands and custom brands
+### Subdomain brands and custom brands
 
 Multibrand orgs have a non-deletable default brand called the subdomain brand. However, you can create several custom brands. The subdomain brand always appears at the Okta subdomain URL and can’t have a custom domain. You can swap out the logo and other assets, but you can’t edit custom code for the sign-in page or error pages. You can only use custom domain and custom code for sign-in pages and error pages on custom brands.
 
-### Multibrand customizations and emails
+### Multibrand and resource sets
+
+Make a custom admin role specific to a brand by using a customization resource type. For more information:
```

Comment:
we need to enable `Redesigned Resource Editor for Admin roles` flag to get the `Select customization` option. I think @grahamsmith-okta will be adding in the actual reference doc. not sure if it is worth mentioning here as well or not

Reply:
@arsalanbadar-okta A good question. We don't mention other flags on this page, so I'm reluctant to add one. I am linking to a page to which Graham is adding the FF :)